### ISSUE TITLE
Log execution processor does not output to stdout

### ISSUE THREAD

**Issue by debunn** [dkron-io/dkron#482]
## Expected Behavior
When using the Log Execution Processor, output from the executed job gets sent to stdout / stderr

## Actual Behavior
No output is sent to stdout / stderr

## Steps to Reproduce the Problem

  1. Create a test batch job on the dkron client (/scripts/test.bash):
  ```
  #!/bin/bash
  echo "This is a test script"
  ```
  2.  Create a test job on the client / server:
  ```
  {
	"name":"test",
        "executor": "shell",
        "executor_config": {
                "command": "/scripts/test.bash",
                "shell": "true"
        },
	"schedule":"@every 2m",
	"concurrency": "forbid",
	"tags":{
		"role":"job"
	},
    "processors": {
        "log": {
            "forward": true
        }
    }
}
  ```
  3. Watch the stdout of the client / server for the output of the job:  "This is a test script".  It does not appear, despite successful execution of the job.
  4. Edit the /scripts/test.bash file to include an additional command which will fail, such as:  "rm /file/does/not/exist".  Neither the echo command (stdout) or the failed delete command (rm /file/does/not/exist -> stderr) will be sent to stdout / stderr.

## Specifications
  - Version: 1.0.1
  - Platform: dkron server (Docker - Alpine Linux 3.8.2), dkron client (CentOS 7.6)
  - Backend store: Consul 1.0.6

Note:  This seems to be a result of the log_output.go Process function not actually sending the Execution.Output of the job to stdout / stderr

**Comment by debunn** [comment]
I'd also argue that for a job that fails, all successfully issued commands (before the error occurred) should be sent to stdout, while the error message should be sent to stderr, to aid in troubleshooting the exact problem with the job

**Comment by victorcoder** [comment]
Thanks for reporting, I don't know where the actual code went, maybe lost in my git stage but the plugin was not doing anything at all.

Fixed in #483

**Comment by debunn** [comment]
Thanks Victor

As mentioned though, would it be possible to always output the stdout, even when an error occurred?  I think this would be useful for troubleshooting what caused the error in the job:

(something like this instead?)
```
	fmt.Printf("----- BEGIN OUTPUT job=%s execution=%s -----\n", args.Execution.JobName, args.Execution.Key())
	fmt.Print(string(args.Execution.Output))
	fmt.Printf("----- END OUTPUT -----\n")
	if !args.Execution.Success {
		fmt.Fprintf(os.Stderr, "----- BEGIN OUTPUT job=%s execution=%s -----\n", args.Execution.JobName, args.Execution.Key())
		fmt.Fprint(os.Stderr, string(args.Execution.Output))
		fmt.Fprintf(os.Stderr, "----- END OUTPUT -----\n")
	}
```
